### DISCUSSION TITLE
How to find opencv.pc after installing OpenCV?

### DISCUSSION THREAD

**Discussion by XiaZixun** [microsoft/vcpkg#39672] [Q&A]
I installed opencv through `vcpkg` and hope to use `pkg-config` to import it into VSCode.

However, I can't find the `opencv.pc` or similarly named file in `~/vcpkg/installed/arm64-osx/lib/pkgconfig`. But I found many pc files for different libraries.

May I ask, does vcpkg not generate pc files when installing opencv? How should I request it to produce the corresponding pc files?
<img width="890" alt="image" src="https://github.com/microsoft/vcpkg/assets/116243141/66371b92-10ae-4842-a5cc-09391dadc835">

**Comment by AugP** [comment]
This sounds like a bug since OpenCV supports pkg-config and our corresponding port for it should as well. I created an issue out of this to track since we track bugs in our issue tracker: https://github.com/microsoft/vcpkg/issues/39833
